e2e/acceptance/crate-dependencies.spec.ts

```diff
@@ -76,16 +76,17 @@ test.describe('Acceptance | crate dependencies page', { tag: '@acceptance' }, ()
     });
 
     await ember.addHook(async owner => {
-      // Load `crate` and then explicitly unload the side-loaded `versions`.
+      // Load `crate` and ensure all `versions` are not loaded.
       let store = owner.lookup('service:store');
       let crateRecord = await store.findRecord('crate', 'foo');
-      let versions = crateRecord.hasMany('versions').value();
-      versions.forEach(record => record.unloadRecord());
+      let versionsRef = crateRecord.hasMany('versions');
+      globalThis.version_ids = versionsRef.ids();
     });
 
     await page.goto('/crates/foo/1.0.0/dependencies');
 
     await expect(page).toHaveURL('/crates/foo/1.0.0/dependencies');
+    await page.waitForFunction(() => globalThis.version_ids?.length === 0);
```

Comment:
what is the reasoning behind this assertion? at first glance I think I'd just remove the whole `ember.addHook()` call above since it's no longer needed 🤔

Reply:
I was trying to do the same assertion as done in QUnit. But yeah, I think we could just remove the assertion for both.